src/videos.md

```diff
@@ -1,4 +1,17 @@
 ---
 title: Videos
 layout: _layouts/Videos.jsx
+videos:
+  - id: 1
+    header: "Header"
+    link: "dQw4w9WgXcQ"
+  - id: 2
+    header: "2"
+    link: "dQw4w9WgXcQ"
```

Comment:
runny rickroll lol, but for now can you switch it to no video and change the headers to "Coming Soon". We are going to deploy this tomorrow so don't want the rick roll showing.